rnote-ui/data/ui/strokecontentpreview.ui

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ### StrokeContentPicker ### -->
+<interface>
+  <template class="RnStrokeContentPreview" parent="GtkWidget">
+    <property name="layout-manager">
+      <object class="GtkBoxLayout">
+        <property name="spacing">6</property>
+        <property name="orientation">vertical</property>
+      </object>
+    </property>
+    <style>
+      <class name="card" />
+    </style>
+    <child>
+      <object class="GtkOverlay" id="preview_overlay">
+        <property name="halign">fill</property>
+        <property name="valign">fill</property>
+        <property name="hexpand">true</property>
+        <property name="vexpand">true</property>
+        <child type="overlay">
+          <object class="GtkBox" id="pages_controls_box">
+            <property name="orientation">horizontal</property>
+            <property name="halign">fill</property>
+            <property name="valign">start</property>
+            <property name="hexpand">true</property>
+            <property name="vexpand">false</property>
+            <property name="margin-start">6</property>
+            <property name="margin-end">6</property>
+            <property name="margin-top">6</property>
+            <property name="margin-bottom">6</property>
+            <child>
+              <object class="GtkBox">
+                <property name="orientation">horizontal</property>
+                <property name="halign">start</property>
+                <property name="hexpand">true</property>
+                <style>
+                  <class name="linked" />
+                </style>
+                <child>
+                  <object class="GtkButton" id="prev_page_button">
+                    <property name="icon-name">left-symbolic</property>
+                    <style>
+                      <class name="osd" />
+                    </style>
+                  </object>
+                </child>
+                <child>
+                  <object class="GtkButton" id="next_page_button">
+                    <property name="icon-name">right-symbolic</property>
+                    <style>
+                      <class name="osd" />
+                    </style>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="GtkEntry" id="page_entry">
+                <property name="halign">end</property>
+                <property name="hexpand">false</property>
+                <property name="input-purpose">digits</property>
+                <property name="max-width-chars">4</property>
+                <property name="xalign">0.5</property>
+                <property name="placeholder-text" translatable="yes">Page</property>
+                <style>
+                  <class name="osd" />
+                </style>
+              </object>
+            </child>
+          </object>
```

Comment:
What about using a `GtkSpinButton` instead of implementing this manually? I guess the increment/decrement symbols would be different, and all page controls would be centered instead of being split up and aligned to the upper corners. 

I'm not sure how it'd affect the UX. Using a spin button (and having a single toolbar at the bottom) *might* look better when adding new elements for e.g. zoom in the future, but the way it is now also looks good and feels intuitive to me.

Reply:
I am unsure if the `+` and `-` buttons fit to navigating between pages. What about leaving it for now, but grouping the navigation buttons and page entry together when/if zooming gets added to keep it balanced visually?

Related: what do you think, should the controls be at the top or at the bottom?

Reply:
Yeah, sounds good to me.

I think as long as they are floating buttons, it's fine if they're at the top. But if we add more buttons and functionality, we should (IMO) consider using a non-floating/non-overlapping toolbar instead, which probably should be at the bottom.

Reply:
yeah I agree. I'll keep it in mind when implementing the follow-up